hugo/content/docs/reference/sql-functions.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "SQL Functions"
+weight: 1
+draft: false
+lastmod: 2020-12-3
+type: docs
+layout: single-col
+---
+Data Connect's SQL dialect has been selected for compatibility with current major open source database platforms including Trino, PostgreSQL, and MySQL, and BigQuery. There are occasional name or signature differences, but a Data Connect implementation atop any of the major database platforms should be able to pass through queries that use the functions listed below with only minor tweaks.
+
+The functions below are a subset of those available in Trino 341. In a conformant Data Connect implementation, these functions must behave according to the Trino documentation. To assist with implementations directly on other database platforms, the [Trino Functions Support Matrix](https://docs.google.com/document/d/1y51qNuoe2ELX9kCOyQbFB4jihiKt2N8Qcd6-zzadIvk) captures the differences between platforms in granular detail. 
```

Comment:
Is this net new or just moved?

I apologize if I missed the discussion but I am hesitant to make the claim that a conformant implementation needs to support everything here...

Reply:
Not net new, should be just lifted from SPEC.md. I'll check.

Reply:
I took a few minutes now to go over the list of functions, and I'm comfortable with the "optional" approach now.